.travis.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+group: stable
+dist: trusty
+sudo: required
+
+language: python
+python:
+  - 2.7
+
+branches:
+  only:
+    - develop
+    - master
+
+env:
```

Comment:
Sorry my bad, I didn't mean for to remove Ansible 2.3 completely, I think we still need to support this version as I know some people are using it:

```YAML
ANSIBLE_INSTALL_VERSION=2.3.3.0
```